ui/src/layouts/Tabs.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+    <BaselineLayout :page-title="$route.meta.title">
+        <div :id="'nrdb-page-' + $route.meta.id" class="nrdb-layout--tabs nrdb-ui-page" :class="page?.className">
+            <div>
+                <!-- Render any widgets with a 'page' scope -->
+                <component
+                    :is="widget.component"
+                    v-for="widget in pageWidgets"
+                    :id="widget.id"
+                    :key="widget.id"
+                    :props="widget.props"
+                    :state="widget.state"
+                />
+            </div>
+
+            <v-tabs v-if="orderedGroups" v-model="tab" show-arrows>
+                <v-tab v-for="t in orderedGroups" :key="t.id" :value="t.id">{{ t.name }}</v-tab>
+            </v-tabs>
+
+            <v-tabs-window v-if="orderedGroups" v-model="tab">
+                <v-tabs-window-item v-for="t in orderedGroups" :key="t.id" :value="t.id">
+                    <div
+                        class="nrdb-ui-group" :class="getGroupClass(t)"
+                        :disabled="t.disabled === true ? 'disabled' : null"
+                        :style="`grid-column-end: span min(${ t.width }, var(--layout-columns)`"
+                    >
+                        <v-card variant="outlined" class="bg-group-background">
+                            <template #text>
+                                <widget-group :group="t" :widgets="widgetsByGroup(t.id)" :row-height="rowHeight" />
+                            </template>
+                        </v-card>
+                    </div>
+                </v-tabs-window-item>
+            </v-tabs-window>
+            <!-- <div v-if="orderedGroups" :id="'nrdb-page-' + $route.meta.id" class="nrdb-layout--grid nrdb-ui-page" :class="page?.className">
```

Comment:
Can we do a tree shaking if this is not being used

Reply:
Yes - absolutely, lazy coding on my part

Reply:
No worries at all 😄